addons/web/static/src/core/signature/name_and_signature.js

```diff
@@ -123,6 +122,19 @@ export class NameAndSignature extends Component {
         this.props.signature.isSignatureEmpty = this.isSignatureEmpty;
     }
 
+    /**
+     * Load a base64 encoded image, as dataURL, into the signature field.
+     * Update the isSignatureEmpty flag to reflect the correct state.
+     * This prevents the flag from incorrectly staying true when a signature
+     * image is already passed in the props.
+     */
+    fromDataURL(dataURL) {
+        this.clear();
```

Comment:
Why this `clear` call ? :thinking:
I see it coming from the original call in the `useEffect` but should stay separate, I think.